dev/release/check-rat-report.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+##############################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
we do have a RAT check from maven, is this check a replacement or covers other usecase?

Reply:
It looks like we somehow missed adding the RAT check in maven. I was also surprised.

Reply:
@comphead I misread your comment. Where is the rat check in maven?

Reply:
nm, I see it now.

We need to run the rat check against the source tarball before we release it. The maven project is checking the contents of the repo not the release tarball, so maybe we need both. I will check how we are doing this in DataFusion.

Reply:
yeah, ideally to remove duplications, perhaps we can remove RAT check in maven then and live only with the release one

Reply:
DataFusion uses the same rat script from CI and from verify-release.sh. 

If we do the same with Comet and remove the maven task it could cause more work for developers because they would not catch issues as part of a standard maven build and would have to run a separate script. I am not sure if it is worth the effort to change this right now.